components/states/CalendarState.jsx

```diff
@@ -42,19 +42,19 @@ const CalendarState = () => {
 
   const sendEvent = (time) => {
     if (state.context.reservationType == "recreational") {
-      send({ type: STATE_ACTIONS.IS_DIVER_CERTIFIED, date, time });
+      send({
+        type: STATE_ACTIONS.IS_DIVER_CERTIFIED,
+        date,
+        time,
```

Comment:
we also probably need to clear this when the user clicks back, we will need to figure out how to do it as well (its ok for in another PR)